src/App.tsx

```diff
@@ -1,9 +1,8 @@
-import React from "react";
-
 import { useState } from "react";
 import reactLogo from "./assets/react.svg";
 import viteLogo from "/vite.svg";
 import "./App.css";
+import Button from "./components/Button/Button";
```

Comment:
avoid relative paths
import Button from "Components/Button";

Reply:
You should add index.js file for every component to resolve like above instead of Components/Button/Button.
https://stackoverflow.com/a/44092481

Reply:
updated for this component , will follow same for all components.